Client/Frontend/Home/JumpBackIn/JumpBackInViewModel.swift

```diff
@@ -248,10 +256,37 @@ private extension JumpBackInViewModel {
         let site = Site(url: itemURL, title: item.displayTitle)
         let descriptionText = site.tileURL.shortDisplayString.capitalized
 
+        let id = Int(arc4random())
+        cell.tag = id
+        var heroImage: UIImage?
+        var favicon: UIImage?
+
+        siteImageHelper.fetchImageFor(site: site,
+                                      imageType: .favicon,
+                                      shouldFallback: false) { image in
+            guard cell.tag == id else { return }
+            favicon = image
+            let cellViewModel = JumpBackInCellViewModel(titleText: site.title,
+                                                        descriptionText: descriptionText,
+                                                        favIconImage: favicon,
+                                                        heroImage: heroImage)
+            cell.configure(viewModel: cellViewModel, theme: self.theme)
+        }
+        siteImageHelper.fetchImageFor(site: site,
+                                      imageType: .heroImage,
+                                      shouldFallback: true) { image in
+            guard cell.tag == id else { return }
+            heroImage = image
+            let cellViewModel = JumpBackInCellViewModel(titleText: site.title,
+                                                        descriptionText: descriptionText,
+                                                        favIconImage: favicon,
+                                                        heroImage: heroImage)
+            cell.configure(viewModel: cellViewModel, theme: self.theme)
+        }
```

Comment:
This could be written much more cleanly, but this will be the first area to be replaced with the new favicon system so I didn't want to waste time on it.